### PR TITLE
Add integration and raw data to README and examples

### PR DESCRIPTION
**Pull request by ekrengel**

### CODE REVIEW

File: nodejs/awsx/apigateway/README.md [hunk 1]
```diff
@@ -94,6 +94,57 @@ let endpoint = new awsx.apigateway.API("example", {
...
+
+```ts
+import * as awsx from "@pulumi/awsx";
+
+let endpoint = new awsx.apigateway.API("example", {
+    routes: [{
+        path: "/integration",
+        target: {
+            uri: "https://www.google.com",
+            type: "http_proxy",
```
Comment (CyrusNajmabadi) [inline nodejs/awsx/apigateway/README.md]: if it's http_proxy, do you need to create a vpc_link: https://docs.aws.amazon.com/apigateway/latest/developerguide/getting-started-with-private-integration.html  ?  or is that optional for an `http_proxy`?

File: nodejs/awsx/apigateway/README.md [hunk 2]
```diff
@@ -94,6 +94,57 @@ let endpoint = new awsx.apigateway.API("example", {
...
+        },
+    }],
+})
+```
+
+#### Raw Data Route
+
+A Raw Data Route is a fallback route for when raw swagger control is desired.  The `data` field should be an object that will be then included in the final swagger specification.
+
```
Comment (CyrusNajmabadi) [inline nodejs/awsx/apigateway/README.md]: do we want to perhaps link to https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-swagger-extensions-integration.html or another useful reference doc for the "x-amazon-apigateway-integration" swagger point?

File: nodejs/awsx/examples/api/index.ts [hunk 3]
```diff
@@ -82,6 +82,30 @@ const api = new awsx.apigateway.API("myapi", {
...
+                "passthroughBehavior": "when_no_match",
+                "httpMethod": "GET",
+                "type": "http_proxy",
+            },
+        },
+    }, {
+        path: "/integration",
+        target: {
+            uri: "https://www.google.com",
+            type: "http_proxy",
```
Comment (CyrusNajmabadi) [inline nodejs/awsx/examples/api/index.ts]: does this actually work?

Comment (ekrengel) [reply nodejs/awsx/examples/api/index.ts]: it does -- it just returns a blob from google

File: nodejs/awsx/apigateway/README.md (hunk 1 above)
Comment (ekrengel) [reply nodejs/awsx/apigateway/README.md]: i think its optional since it works as is

**Comment by CyrusNajmabadi** [comment]
LGTM.  this might be another area where we mght benefit from helpers *in the future* to allow people to easily make integrations to services.

**Pull request merged**